rewrites/src/main/scala/fix/scala213/Any2StringAdd.scala

```diff
@@ -0,0 +1,33 @@
+package fix.scala213
+
+import scalafix.v1._
+
+import scala.meta._
+
+object Any2StringAdd {
+  val plusString = SymbolMatcher.exact(
+    "scala/Predef.any2stringadd#`+`().",
+    "scala/Byte#`+`().",
+    "scala/Short#`+`().",
+    "scala/Char#`+`().",
+    "scala/Int#`+`().",
+    "scala/Long#`+`().",
+    "scala/Float#`+`().",
+    "scala/Double#`+`().",
+  )
+}
+
+final class Any2StringAdd extends SemanticRule("fix.scala213.Any2StringAdd") {
+  import Any2StringAdd._
+
+  override def fix(implicit doc: SemanticDocument): Patch = {
+    doc.tree.collect {
+      case plusString(Term.ApplyInfix(lhs, _, _, _)) => addToString(lhs)
+    }.asPatch
+  }
+
+  private def addToString(term: Term) = term match {
+    case _: Term.Name | _: Term.Select | _: Term.Block => Patch.addRight(term, ".toString")
+    case _ => Patch.addLeft(term, "(") + Patch.addRight(term, ").toString")
```

Comment:
my only concern here is that it unnecessarily boxes primitives.

Reply:
Ah, that must be why the deprecation message for these is to use string interpolation.

Should we do `"" + x` for these?  (And `"" + (...)` for non-Name/Select/Block.) . Or should we try to rewrite it to use string interpolation?

Reply:
I don't know. IIRC, string interpolation is special-cased by the compiler in 2.13, so it's probably at least as efficient as other options, if not more so?

Reply:
Yeah it's the right choice.  But for mechanical rewrites you could get some weird rewrites, so maybe this is best?

Reply:
> that must be why the deprecation message for these is to use string interpolation.

> IIRC, string interpolation is special-cased by the compiler in 2.13


Yes. see https://github.com/scala/scala/pull/7201 https://github.com/scala/bug/issues/11025